src/test/java/benchmark/AddError.java

```diff
@@ -28,7 +28,7 @@ public class AddError {
     @BenchmarkMode(Mode.SingleShotTime)
     @Warmup(iterations = 1, batchSize = 50000)
     @Measurement(iterations = 1, batchSize = 5000)
-    public ExecutionContext benchMarkAddError() {
+    public ExecutionContext benchMarkAddError() throws InterruptedException {
```

Comment:
why do we need to add these exceptions to every method signature?

Reply:
The generated jmh test class uses these methods and it is wrapped inside a try statement which catches InterruptedException. When I build the jmh test jar it throws compilation error. In order to fix it I had to add it to all the methods.

Reply:
I didn't need to do that last time. Maybe you need the additional gradle changes I mentioned here: https://github.com/graphql-java/graphql-java/pull/2871#issuecomment-1164909737